app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file, new_file = command.split()
```

Comment:
You need before to validate that command.split() have exactly 3 parameters

Reply:
If that have more or less?

Reply:
There should be exactly 3 parameters here